src/chrome/content/rules/FreeDesktop.xml

```diff
@@ -1,96 +1,58 @@
-
-		- planet *
-
-	* Shows www.freedesktop.org
-
-
-	Problematic hosts in *freedesktop.org:
-
-		- download ¹
-		- sitewranglers ¹
-		- xorg ¹
-
-	¹ Shows www.freedesktop.org, preemptable redirect
-
-
-	Mixed content:
-
-		- css on freetype from fonts.googleapis.com ˢ
-		- Bug on freetype from api.flattr.com ˢ
-
-	ˢ Secured by us, see https://www.paulirish.com/2010/the-protocol-relative-url/
-
+	Mismatched:
+		- download
+		- sitewranglers
+		- tango
 -->
 <ruleset name="freedesktop.org (partial)">
-
-	<!--	Direct rewrites:
-				-->
 	<target host="freedesktop.org" />
+	<target host="www.freedesktop.org" />
 	<target host="apoc.freedesktop.org" />
 	<target host="bugs.freedesktop.org" />
 	<target host="bugzilla.freedesktop.org" />
 	<target host="cgit.freedesktop.org" />
-	<target host="dri.freedesktop.org" />
 	<target host="dbus.freedesktop.org" />
+	<target host="download.freedesktop.org" />
+	<target host="dri.freedesktop.org" />
 	<target host="fontconfig.freedesktop.org" />
 	<target host="freetype.freedesktop.org" />
 	<target host="gstreamer.freedesktop.org" />
 	<target host="gypsy.freedesktop.org" />
+	<target host="hal.freedesktop.org" />
+	<target host="icon-theme.freedesktop.org" />
 	<target host="ldtp.freedesktop.org" />
 	<target host="libbsd.freedesktop.org" />
 	<target host="libdlo.freedesktop.org" />
 	<target host="liboil.freedesktop.org" />
 	<target host="libopenraw.freedesktop.org" />
+	<target host="libspectre.freedesktop.org" />
 	<target host="lists.freedesktop.org" />
 	<target host="nice.freedesktop.org" />
 	<target host="nouveau.freedesktop.org" />
 	<target host="patchwork.freedesktop.org" />
 	<target host="people.freedesktop.org" />
 	<target host="piglit.freedesktop.org" />
+	<target host="planet.freedesktop.org" />
 	<target host="pm-utils.freedesktop.org" />
 	<target host="poppler.freedesktop.org" />
 	<target host="portland.freedesktop.org" />
 	<target host="secure.freedesktop.org" />
+	<target host="sitewranglers.freedesktop.org" />
 	<target host="specifications.freedesktop.org" />
 	<target host="standards.freedesktop.org" />
 	<target host="swfdec.freedesktop.org" />
 	<target host="telepathy.freedesktop.org" />
-	<target host="xcb.freedesktop.org" />
 	<target host="wayland.freedesktop.org" />
 	<target host="wiki.freedesktop.org" />
-	<target host="www.freedesktop.org" />
-
-	<!--	Complications:
-				-->
-	<target host="download.freedesktop.org" />
-	<target host="sitewranglers.freedesktop.org" />
+	<target host="xcb.freedesktop.org" />
 	<target host="xorg.freedesktop.org" />
 
-		<test url="http://people.freedesktop.org/~airlied/" />
-		<test url="http://xcb.freedesktop.org/dist" />
+	<test url="http://people.freedesktop.org/~airlied/" />
 
+	<securecookie host=".+" name=".+" />
 
-	<securecookie host="^bug" name=".+" />
-
-
-	<!--	Redirects as so:
-				-->
 	<rule from="^http://(?:download|sitewranglers)\.freedesktop\.org/"
 		to="https://fontconfig.freedesktop.org/" />
```

Comment:
For both `download` and `sitewranglers` I get `302` to `https://www.freedesktop.org/wiki/` . The content is also different from `fontconfig`, this rule might need a rewrite.

Reply:
I'm getting a server side redirect to https. Removing this rule completely now.